R/load_flu_hosp_data.R

```diff
@@ -91,7 +98,9 @@ load_flu_hosp_data <- function(as_of = NULL,
       jsonlite::fromJSON()
     csv_path <- tail(temp$archive_link$url, 1)
     data <- readr::read_csv(csv_path)
-    state_dat <- data %>%
+    ## value depends on which pathogen
```

Comment:
I am wondering if we could reduce some code duplication by conditioning inside of the transmute? Am I missing something here?

```suggestion
    ## value depends on which pathogen
    state_dat <- data %>%
        dplyr::transmute(
          geo_value = tolower(state),
          date = date - 1,
          epiyear = lubridate::epiyear(date),
          epiweek = lubridate::epiweek(date),
          value = ifelse(
            pathogen == "flu",
            previous_day_admission_influenza_confirmed,
            previous_day_admission_adult_covid_confirmed + previous_day_admission_pediatric_covid_confirmed)
        ) %>%
        dplyr::arrange(geo_value, date)
```

Reply:
Agree, but I tried to do this for over 30 minutes when originally writing it and didn't get it to work, so I gave up.